UKHomeOffice/sgmr-service
# SPCR-108: Update aria labelling on GOV UK Logo

## Description
Fix aria and screen reader labelling for GOV.UK logo

## To Test
* Go to the landing page (or any other page that is part of the sPCR service)
* Enable a screen reader
* Select the GOV.UK logo or text in the top left
* The screen reader should read the GOV.UK label

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
